audformat/core/utils.py

```diff
@@ -64,18 +65,35 @@ def concat(
     or one column contains ``NaN``.
     If ``overwrite`` is set to ``True``,
     the value of the last object in the list is kept.
+    If ``overwrite`` is set to ``False``,
+    a custom aggregation function can be provided
+    with ``aggregate_function``
+    that converts the overlapping values
+    into a single value.
 
     Args:
         objs: objects
         overwrite: overwrite values where indices overlap
+        aggregate_function: function to aggregate values
+            for all entries
+            that contain more than one value per index.
+            The function gets a dataframe row as input.
```

Comment:
```suggestion
            The function gets a :class:`pd.DataFrame` row as input.
```

But maybe more explicit:

```
gets a :class:`pd.Series` with the overlapping values as input
```

?

Reply:
Good idea, I changed to your second suggestion.